polaris-react/src/components/IndexTable/IndexTable.stories.tsx

```diff
@@ -1738,6 +1749,7 @@ export function WithRowStatus() {
   return (
     <LegacyCard>
       <IndexTable
+        condensed={useBreakpoints().smDown}
```

Comment:
IndexTable cuts the content when condensed:
<img width="1494" alt="image" src="https://github.com/Shopify/polaris/assets/104942025/41d3a0d1-bbd3-4885-8be9-95d53c2e396b">

Reply:
The index table is side scrollable, this is intentional vs allowing content wrapping